templates/etc/apt/sources.list.j2

```diff
@@ -27,7 +27,7 @@
 {%     endfor %}
 {%   else %}
 {%     for entry_uri in (entry.uri|d([]) + entry.uris|d([])) %}
-{%       if entry_uri in apt__tpl_seen_sources %}
+{%       if (entry_uri in apt__tpl_seen_sources and apt__sources_filter_duplicates|bool) %}
```

Comment:
The problem is that an "entry" can have multiple URLs to mirrors, so parsing that would be complicated, and involve multiple `for` loops, for little payoff. This functionality is mostly to avoid an issue when the original mirror is the same that the default mirrors - it's a cheap way to avoid warning emails.